### PR TITLE
POWHEG+JHUGen mass scan cards for 2017 MC

### PR DESCRIPTION
**Pull request by hroskes**
Each one with a template, a script to make the cards, and a gitignore

### CODE REVIEW

File: bin/Powheg/production/2017/13TeV/HWJ_NNPDF31_13TeV/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+HW*J_HanythingJ_NNPDF31_13TeV_M*.input
```
Comment (perrozzi) [inline bin/Powheg/production/2017/13TeV/HWJ_NNPDF31_13TeV/.gitignore]: I don't think this file should be committed

File: bin/Powheg/production/pre2017/13TeV/HZJ_NNPDF30_13TeV/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+HZJ_HanythingJ_NNPDF30_13TeV_M*.input
```
Comment (perrozzi) [inline bin/Powheg/production/pre2017/13TeV/HZJ_NNPDF30_13TeV/.gitignore]: neither this

**Comment by perrozzi** [comment]
the rest looks ok to me

**Comment by hroskes** [comment]
@perrozzi 
I was putting those there so that when you run the script and then do git status, you don't see all the cards there (which are not supposed to be committed).  Is that a problem?

**Pull request merged**